oeps/oep-0011-bp-FED-technology.rst

```diff
@@ -48,33 +48,42 @@ Specification
 Technology Selection
 ~~~~~~~~~~~~~~~~~~~~
 
-1. **Server-side content should be rendered with Django Templates**
+1. **Use React and Redux**
```

Comment:
Reminder to add a link to the Bootstrap OEP since Bootstrap is another technology we've adopted.